.github/workflows/ci_eval.yaml

```diff
@@ -36,17 +36,15 @@ jobs:
         shell: bash
     env:
       PIP_CACHE_DIR: "${{ github.workspace }}/.pip-cache"
-      SHARK_PLATFORM_REPO_ROOT: ${{ github.workspace }}
     steps:
+      - uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v4.2.2
```

Comment:
Just seeing this now, there is another job in this workflow that needs to be updated as well. Maybe in another PR.